### DISCUSSION TITLE
is v1.5.0-rc.2 support nftables ?

### DISCUSSION THREAD

**Discussion by BobSquarePants** [evilsocket/opensnitch#609] [Q&A] [answered]
Hi,

I'll will migrate my system to [nftables](https://www.netfilter.org/projects/nftables/index.html)
Therefore get rid of iptables.

is v1.5.0-rc.2 support nftables ? not to create new nftables rules etc.. but if the installation process will create a tables\rules ? to allow opensnitch to catch packet before they go out.

Thanks

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
yes, nftables is supported for traffic interception. If you delete completely iptables from your system, opensnitch will use nftables.
You can also change it in the configuration file `/etc/opensnitchd/default_config.json`

If you had [system rules](https://github.com/evilsocket/opensnitch/wiki/System-rules) added to `/etc/opensnitchd/system-fw.json` then no, nftables is not supported yet.
